### PR TITLE
Fixed error in import solution notebook

### PR DESCRIPTION
**Pull request by jyaistMap**
Added underscores to the zipfile name in creating the item.

### CODE REVIEW

File: labs/import_data.ipynb [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "\n",
     "If you have not completed the [Download data lab](https://developers.arcgis.com/labs/python/download-data), the code in the **The Download Data Lab** cell immediately below must be run before the **Import Data Lab** cells will run. These cells contain the code in the ArcGIS API for Python [Download data lab solution](download_data.ipynb). The LA_Hub_Datasets.zip directory from which you import data is only available in a single notebook.esri.com session.\n",
     "\n",
-    "If you have completed the [Download data lab](https://developers.arcgis.com/labs/python/download-data) and are still in that notebooks.esri.com session, skip to [The Import Data Lab](#import_section) section of this lab."
+    "If you have completed the [Download data lab](https://developers.arcgis.com/labs/python/download-data) and are still in that notebook.esri.com session, skip to [The Import Data Lab](#import_section) section of this lab."
```
Comment (rohitgeo) [inline labs/import_data.ipynb]: notebooks.esri.com is the correct URL, not notebook.esri.com. It should probably say "are still in that notebook, skip to..."

Comment (rohitgeo) [reply labs/import_data.ipynb]: "notebook.esri.com" is referred to twice in the note.

File: labs/import_data.ipynb [hunk 2]
```diff
@@ -41,24 +41,25 @@
     }
    ],
    "source": [
-    "from arcgis.gis import GIS\n",
     "import getpass\n",
     "\n",
     "password = getpass.getpass(\"Enter password, please: \")\n",
-    "dev_gis = GIS('https://arcgis.com', 'username', password)"
+    "dev_gis2 = GIS('https://arcgis.com', 'username', password)"
```
Comment (rohitgeo) [inline labs/import_data.ipynb]: Suggest renaming the "dev_gis2" to just "gis"

**Comment by jyaistMap** [comment]
Thanks, @rohitgeo.

**Pull request merged**